lane.py

```diff
@@ -269,9 +289,17 @@ def _from_request(
         if isinstance(entrypoint, ProblemDetail):
             return entrypoint
         entrypoint, is_default = entrypoint
+
+        max_cache_age = get_argument(
+            Facets.MAX_CACHE_AGE_NAME, None
+        )
+        max_cache_age = cls.load_max_cache_age(max_cache_age)
+        if isinstance(max_cache_age, ProblemDetail):
```

Comment:
The `load_max_cache_age` method can return `None`, but is that an instance of a `ProblemDetail`?

Reply:
I see how the tests make `load_max_cache_age` return a ProblemDetail, but I don't see the actual function returning a ProblemDetail, just for a value or None.

Reply:
No, in the current version `load_max_cache_age` will never return `ProblemDetail`. But when calling code that loads data from the HTTP request I find it useful to consistently use this pattern:

```
loaded = load_data()
if isinstance(loaded, ProblemDetail):
   return loaded
````

That way if I change `load_data` later so it _does_ return a ProblemDetail, the caller is already ready and tested. In this case we might change `load_max_cache_age` to return a ProblemDetail (rather than None) if you pass in something that's not a number.